srv/methods.go

```diff
@@ -548,6 +548,20 @@ func (s *APIServer) getSyncStatus(_ context.Context, data json.RawMessage) inter
 	return ResultGetSyncStatus{Sync: s.Node.GetCurrentSync(), Current: int32(heights.DirectoryBlock)}
 }
 
+func (s *APIServer) getGraded(ctx context.Context, data json.RawMessage) interface{} {
+	params := ParamsGetGraded{}
+	_, _, err := validate(data, &params)
+	if err != nil {
+		return err
+	}
+
+	result, err := s.Node.Pegnet.SelectGraded(ctx, params.Height)
```

Comment:
You have params.Height as an optional omitempty json field but calling it with a height of 0 never makes sense (1 could happen on a testnet). Both `get-bank` and `get-pegnet-rates` will take the most recently synced height when supplied with a height of 0:
```go
	if params.Height == 0 {
		synced, err := s.Node.Pegnet.SelectSynced(ctx, s.Node.Pegnet.DB)
		if err != nil {
			return err
		}
		params.Height = int32(synced.Synced)
	}
```